src/test/java/hudson/plugins/ec2/AmazonEC2CloudTest.java

```diff
@@ -116,6 +118,32 @@ public void testSshCredentials() throws IOException {
         assertThat(actual.resolvePrivateKey(), notNullValue());
     }
 
+    /**
+     * Ensure that EC2 plugin can use any implementation of SSHUserPrivateKey (not just the default implementation, BasicSSHUserPrivateKey).
+     *
+     * See <a href="https://issues.jenkins-ci.org/browse/JENKINS-63986">JENKINS-63986</a>.
+     */
+    @Test
```

Comment:
```suggestion
    @Issue("JENKINS-63986")
    @Test
```
Would probably need an import as well

Reply:
done